lib/src/base-styles/_nx-tabs.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ /*
+  #Tabs
+*/
+.nx-tab-list {
+  border-bottom: $nx-border-row-dark;
+  padding-left: 0;
+}
+
+.nx-tab {
+  display: inline-block;
+  list-style: none;
+  margin-bottom: -1px;
+  padding: 9px 15px;
```

Comment:
Use our standardized spacing variables for padding

Reply:
2741b82